class_generator/class_generator.py

```diff
@@ -73,7 +71,7 @@ def _is_kind_and_namespaced(client: str, _key: str, _data: Dict[str, Any]) -> Di
     return not_resource_dict
 
 
-def map_kind_to_namespaced(client: str):
+def map_kind_to_namespaced(client: str, newer_cluster_version: bool, schema_definition_file):
```

Comment:
`schema_definition_file` missing type and return type

Reply:
Done